AWS/aws_redshift_get_daily_product_costs.ipynb

```diff
@@ -0,0 +1,1241 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "d25c9341-1533-4a1f-9a1a-8aec41495c85",
+   "metadata": {
+    "deletable": false,
+    "editable": false,
+    "jupyter": {
+     "source_hidden": true
+    },
+    "orderProperties": [],
+    "tags": [
+     "unSkript:nbParam"
+    ],
+    "title": "unSkript Internal"
+   },
+   "outputs": [],
+   "source": [
+    "import json\n",
```

Comment:
The first cell needs to be removed from the runbook. @jayasimha-raghavan-unskript please share the script to sanitize the runbooks.